WordPress/Classes/Services/SuggestionService.swift

```diff
@@ -8,37 +8,28 @@ extension NSNotification.Name {
     public static let suggestionListUpdated = NSNotification.Name.suggestionListUpdated
 }
 
-@objc public class SuggestionService: NSObject {
+/// A service to fetch and persist a list of users that can be @-mentioned in a post or comment.
+class SuggestionService {
 
-    // NSCache works with classes such as NSArray, not structs such as Suggestion or [Suggestion].
-    private class Wrapper<T>: NSObject {
-        let value: T
-        init(_ value: T) {
-            self.value = value
-        }
-    }
-
-    private let suggestionsCache = NSCache<NSNumber, Wrapper<[Suggestion]>>()
+    private let suggestionsCache = NSCache<NSNumber, NSArray>()
     private var siteIDsCurrentlyBeingRequested = [NSNumber]()
 
     private static let shared = SuggestionService()
```

Comment:
Is there a reason we don't just make this internal, instead of returning it in a separate function?

Reply:
Good point! I think I kept this too close to the original implementation of Obj-C.